outputs.tf

```diff
@@ -9,33 +9,29 @@ output "fabric_connection_name" {
 }
 
 output "fabric_connection_status" {
-  description = "Connection provisioning status."
-  value       = module.equinix-fabric-connection.primary_connection.status
+  description = "Connection overall state."
+  value       = module.equinix-fabric-connection.primary_connection.state
 }
 
-output "fabric_connection_provider_status" {
```

Comment:
Converting to draft so I can go through more of these changes and see if they also can/should be modified to match the old outputs